### ISSUE TITLE
link to cached copies of papers; host them where?

### ISSUE THREAD

**Issue by david415** [applied-mixnetworks/mixbib#17]
many of the original paper links are broken. we must not cache the papers on github because they will eventually take them down. we should host the cached copies on our own infrastructure. a cheap VPS would be good enough.

**Comment by meejah** [comment]
"Someone" has been putting many of them in my Tahoe-LAFS grid -- I am fine if we publish the read-caps for these, but of course we don't actually have a way for non-grid participants to download something. I don't really want to make the Introducer fURL for my grid completely public -- but we could use a VPS to host a Web-frontend that has the read-caps. Then, we could publish the readcaps themselves (for anyone who happens to have grid-access) and also publish public URIs that read-through to the grid, *and* just cache the data on the VPS too?

So publish: a URI that's a proxy for a read-cap to the meejah.ca grid; a URI that's a local cached copy; and the actual read-cap.

**Comment by burdges** [comment]
I suppose github might take down sci-hub links?  We're fairly comfortable hosting papers published by anyone at https://gnunet.org/bibliography because we're actually an academic project.  I can look into importing them if you like.

I think anyone who asked their lawyers to send us a nasty legal letter would regret it within a few years, well gnunet was created for peer-to-peer file sharing research.  We'd take down anything requested of course, but doing so might light a fire under some people's asses.  ;)

**Comment by david415** [comment]
I think it's fine to host links to papers on github. But if we host the papers on github then github will act like a company and take them down upon the first complaint they get... no doubt.

**Comment by david415** [comment]
@burdges yes that sounds great. if gnunet hosts the papers we'll link to them from mixbib.
